app-text/xapers/xapers-0.8.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6})
+
+inherit distutils-r1
+
+DESCRIPTION="Xapers is a personal document indexing system."
+HOMEPAGE="https://finestructure.net/xapers/"
+SRC_URI="https://finestructure.net/xapers/releases/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~s390 ~sparc ~x86"
```

Comment:
were you able to test the package on all of these arches?

Reply:
Going by the comment, I should only keyword arches I've actually tested it on, so I've dropped this down to ~amd64.